setup.cfg

```diff
@@ -23,6 +23,7 @@ install_requires =
     memoization
 python_requires = >=3.8
 include_package_data = True
+zip_safe = False
```

Comment:
Is this line necessary? Apparently this is deprecated and we're probably OK with this being true since we don't have C plugins?

Reply:
> it's a recommendation from mypy, because it won't unzip a zipped dependency to check the typing

Reply:
https://mypy.readthedocs.io/en/stable/installed_packages.html